src/test/resources/cassandra-reaper.yaml

```diff
@@ -30,6 +30,7 @@ logging:
     org.eclipse.jetty: INFO
   appenders:
     - type: console
+      logFormat: "%-6level [%d{HH:mm:ss.SSS}] [%t] %logger{5} - %X{code} %msg %n"
```

Comment:
%X{code} is an MDC thing. "code" is a key which we never populate with a value, so that shouldn't be there.